source/AI.cpp

```diff
@@ -321,6 +321,16 @@ void AI::IssueShipTarget(const PlayerInfo &player, const shared_ptr<Ship> &targe
 
 
 
+void AI::IssueAsteroidTarget(const PlayerInfo &player, const shared_ptr<Minable> &targetAsteroid)
+{
+	Orders newOrders;
+	newOrders.type = Orders::MINING;
+	newOrders.targetAsteroid = targetAsteroid;
+	IssueOrders(player, newOrders, "focusing fire on " + targetAsteroid->Name() + " asteroid.");
```

Comment:
Note that we have minables with names like:
`plant`
`plant2`
`planet cluster`
`large plant`
`large plant2`
`large plant cluster`
`live crystal`
`space flora`
`large space flora`
Whenever we get to adding `Body::noun`, we should already add display names for minables.

Reply:
Should we worry about this now to address it here or is this something that can happen in a follow up PR?

Reply:
Separate PR.